logs_http.module

```diff
@@ -152,6 +146,27 @@ function logs_http_register_event(array $log_entry) {
   // Remove empty values, to prevent errors in the indexing of the JSON.
   $event = logs_http_array_remove_empty($event);
 
+  if (variable_get('logs_http_debugging_mode', FALSE)) {
+    if (!isset($events['debugging'])) {
+      $debugging_event = $event;
+      unset($debugging_event['type']);
```

Comment:
> But this module should provide a useful default so that people can start using it without having to write a custom module.

I agree, so what I think we need to do is:

* Add a hook
* Implement that hook and apply your logic inside ``if (variable_get('http_logs_debug_enviorement', FALSE) {``